tools/acorn-optimizer.mjs

```diff
@@ -1750,16 +1751,27 @@ function reattachComments(ast, commentsMap) {
 
 let suffix = '';
 
-const argv = process.argv.slice(2);
-
-function getArg(arg) {
-  const index = argv.indexOf(arg);
-  if (index == -1) {
-    return false;
-  }
-  argv.splice(index, 1);
-  return true;
-}
+const {
+  values: {
+    'closure-friendly': closureFriendly,
+    'export-es6': exportES6,
+    verbose,
+    'no-print': noPrint,
+    'minify-whitespace': minifyWhitespace,
+    outfile,
+  },
```

Comment:
Is this complex destructuring expression how folks normally use this or just a way to make this change with minimal diff?

If its the latter perhaps we could followup with another change?

Reply:
It looks like the docs do it like this:

```
const { parseArgs } = require('node:util');
const args = ['-f', '--bar', 'b'];
const options = {
  foo: {
    type: 'boolean',
    short: 'f',
  },
  bar: {
    type: 'string',
  },
};
const {
  values,
  positionals,
} = parseArgs({ args, options });
console.log(values, positionals);
```

Where all your options env up on the `values` object rather than in globals.  I think that could make sense as a followup perhaps.   I have complex destructuring a little non-obvious to read.. but maybe I just need to catch up.